package.json

```diff
@@ -54,7 +54,7 @@
     "cookie": "^0.5.0",
     "dotenv": "^16.3.1",
     "framer-motion": "^10.16.4",
-    "fuels": "0.57.0",
+    "fuels": "^0.67.0",
```

Comment:
Why is this not bumped to the latest v0.69.1

Reply:
it's the version used on the latest wallet release, this package only gets used for the wallet docs